### PR TITLE
remove RH _M16A2 + _M16A2GL Replace with GST M16A2

### PR DESCRIPTION
**Pull request by Parasite42**
Since I doubt that Robert Hammer weapon packs are used anymore (by most communities) I propose that we replace _M16A2,  _M16A2GL and _M16A3 (as well as add _M16A3GL) with GSTAVO's weapons. I've also added a section in the event any more GSTAVO weapon packs are added. another change is the removal of the RHS links from old RH m16s to RHS m16a4s.

### CODE REVIEW

File: loadouts/content/broma-weapon-list.sqf [hunk 1]
```diff
@@ -551,9 +551,6 @@ _M110 = ["RH_m110",_20rnd762M110];
 
 _M16A1 = ["RH_m16a1",_30rnd556];
 _M16A1GL = ["RH_m16a1gl",_30rnd556,_40mm];
-_M16A2 = ["RH_m16a2",_30rnd556];
-_M16A2GL = ["RH_m16a2gl",_30rnd556,_40mm];
-_M16A3 = ["RH_m16a3",_30rnd556];
```
Comment (Coryf88) [inline loadouts/content/broma-weapon-list.sqf]: Don't remove these.

File: loadouts/content/broma-weapon-list.sqf [hunk 2]
```diff
@@ -869,9 +866,6 @@ _PKM = ["rhs_weap_pkm",_100rnd762PKP];
 
 _M16A1 = _RHSM16A4;
 _M16A1GL = _RHSM16A4GL;
-_M16A2 = _RHSM16A4;
-_M16A2GL = _RHSM16A4GL;
-_M16A3 = _RHSM16A4;
```
Comment (Coryf88) [inline loadouts/content/broma-weapon-list.sqf]: Don't remove these.

**Comment by Parasite42** [comment]
I'll rename the gstavo lines to _GST**** and re-add the RH and RHS lines. I can't really change anything effectively on  my phone, so I'll change it at home. Thanks again Cory.

**Comment by Coryf88** [comment]
No, don't prefix the GSTAVO variable names, the way they are is fine.

**Comment by Parasite42** [comment]
Readded.

**Pull request merged**